config/default.js

```diff
@@ -226,6 +227,9 @@ module.exports = {
       liquidityProvider:
         process.env.LIQUIDITY_PROVIDER_KEY ||
         '0xfbc1ee1c7332e2e5a76a99956f50b3ba2639aff73d56477e877ef8390c41e0c6',
+      synchronizer:
+        process.env.SYNCHRONIZER_KEY ||
```

Comment:
Well we shouldn't need, but nf3 expects one. I thought to change nf3 to somehow allow starting without one but since the SDK will be refactored anyway, that'd probably only create conflicts.

Reply:
Does it need to be a valid key? can we put garbage, or an empty variable?